### ISSUE TITLE
Can't add external results with Composition API

### ISSUE THREAD

**Issue by louis-kevin** [vuelidate/vuelidate#1147]
**Describe the bug**
When adding external error to vuelidate, it does not add into field<br>
One of the examples I as able to add only to fields that have rules<br>
In other example I just can't add at all <br>
I do not tested in Options API

**Reproduction URL**
[This just does not work at all](https://codesandbox.io/s/vuelidate-external-errors-in-child-errors-forked-unofcp?file=/src/App.vue)

[This does not work for input without rules](https://codesandbox.io/s/vuelidate-external-errors-in-child-errors-forked-u6zmex?file=/src/App.vue)

**Comment by pm-consultancy** [comment]
Hi Louis-kevin,
I've run in the same problem as you, and is most likely by design
workaround for this is:

```ts
const externalServerValidation = () => true

const rules = {
field: { externalServerValidation},
otherField: { externalServerValidation }
}

const state = {
field: null, 
otherField: null}

const v$ = useVuelidate(rules, state, { $external..... you probally know what to do next})
```

**Comment by louis-kevin** [comment]
Thanks for the response @PaulMenheere 
It seems to work, but It will me better if the external validation could add keys so we don't need to add inputs that does not have a rule in front end

Also, I stamble in this problem when I was trying to create components with default rules, but even if I add the workaround, it seems that the child component does not have the error and only the parent vuelidate has the error

I created this [example](https://codesandbox.io/s/vuelidate-external-errors-in-child-errors-forked-oij4wz?file=/src/components/Child.vue), to reproduce the error

I don't really know if its by design, a bug or a new feature 

Can you guys enlighten me @dobromir-hristov @shentao ?

**Comment by louis-kevin** [comment]
Here a code example: 

```javascript
// Child.vue
const props = defineProps({ modelValue: {} });

const inputChild = ref(props.modelValue);

const rules = {
  inputChild: {
    required,
  },
};

const v = useVuelidate(rules, { inputChild });

// The main idea here is: this vuelidate instance receive the external validation from parent vuelidate instance
```

```javascript
// Parent.vue
const state = reactive({
  inputInParent: "asdfsd",
  inputChild: "asdf",
});
const $externalResults = reactive({});

const externalServerValidation = () => true;

const rules = {
  inputInParent: { required },
  inputChild: { externalServerValidation }, // Better if I don't need to declare the child rules
};
const v = useVuelidate(rules, state, { $externalResults });

// validate method
async function validate() {
  if (!(await v.value.$validate())) return;

  const errors = {
    inputInParent: "Error in inputInParent",
    inputChild: "Error in input in child",
  };
  Object.assign($externalResults, errors);
}
```

**Comment by melishev** [comment]
I encountered the same problem. The error from the server is written to externalResults and passed to v$.

In the developer console I can see the following:
<img width="268" alt="Снимок экрана 2023-01-12 в 17 36 49" src="https://user-images.githubusercontent.com/58311421/212126360-b9a2c18c-dea5-45fb-9be2-70d27d07f6ac.png">

That is, the error is passed to the field, but it is not passed to the overall form error-state

**Comment by k3z** [comment]
In `<script lang="ts" setup>`
with `const v$ = useVuelidate(rules, state)`
return a `ref()`

try `const isFormCorrect = await v$.value.$validate()`

**Comment by kirkbushell** [comment]
Yup, there seems to be a bug with it not updating the state of the errors for validation, even though it's there. The docs are simply wrong currently.

**Comment by kirkbushell** [comment]
So I found an interesting thing - externalResults errors go to $silentErrors. This isn't mentioned anywhere in the documentation. So you have to check boith $errors and $silentErrors (for reasons unbeknownst to me) to get the error output.

**Comment by mostafa-A48** [comment]
I have the same problem and used `const externalServerValidation = () => true`
